modules/auxiliary/scanner/http/owa_login.rb

```diff
@@ -253,7 +255,8 @@ def try_user_pass(opts)
         headers['Cookie'] = 'PBack=0;' << res.get_cookies
       else
         # Login didn't work. no point in going on, however, check if valid domain account by response time.
-        if elapsed_time <= 1
+        # Added check for default value (0), since elapsed_time is not measured if AUTH_TIME is set to "false"
+        if (elapsed_time > 0) && (elapsed_time <= 1)
```

Comment:
Instead of initializing the variable where we need a special conditional check for a specific value, we can add a check to see if the variable has been initialized at all:
```suggestion
        if elapsed_time && elapsed_time <= 1
```

All integer values are truthy in Ruby, so the conditional short-circuits when `elapsed_time` is `nil` and we won't get the exception you were seeing.

Reply:
Hi @acammack-r7,
thank you for your review and the explanation.

I understand that the change you proposed would work if "elapsed_time" were used only inside the "if" clauses.
However, it is also used during most of the "print" statements inside the module (e.g. lines 223, 238, 271 and 320).
Therefore it needs to be defined.

One alternative would be to preface all other statements referring to "elapsed_time" with a check. In my opinion this implementation would be inefficient and thus I did not propose it.

Reply:
There would need to be no extra checks, since interpolating `nil` converts it to a string which is a defined behavior in Ruby that works well for this case:
```
$ irb
irb(main):001:0> "foo #{nil}"
=> "foo "
irb(main):002:0> nil.to_s
=> ""
```

The result would be that when a user doesn't enable `AUTH_TIME` there would be no timing information in the status messages, which is what we want.

Reply:
Ok, thank you for the clarification.
I've updated the pull request.